docs/web/sdk-component-styling.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Component Styling
+description: Geocortex Web - Learn how to style a custom component
+hide_table_of_contents: true
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Geocortex Web Components support fully custom styling using CSS. However, by using Geocortex Web UI Elements, you can take advantage of styled, well tested, UI components that already participate in the [application theme](configuration-theme.mdx).
+
+<Tabs
+  defaultValue="component"
+  values={[
+    { label: "Component", value: "component" },
+    { label: "CSS", value: "css" },
+    { label: "UI", value: "ui" },
+  ]}>
+<TabItem value="component">
+
+```tsx
+import React from "react";
+import {
+    LayoutElement,
+    LayoutElementProperties,
+} from "@geocortex/web/components";
+import List from "@geocortex/web/ui/list";
+import ListItem from "@geocortex/web/ui/list-item";
+import TitleBar from "@geocortex/web/ui/title-bar";
+import Button from "@geocortex/web/ui/button";
+import classNames from "classnames";
+import ExampleComponentModel from "./ExampleComponentModel";
+import "./ExampleComponent.css";
+
+export default function ExampleComponent(
+    props: LayoutElementProperties<ExampleComponentModel>
+) {
+    return (
+        <LayoutElement {...props}>
+            <List dense={true}>
+                <TitleBar>Some Title</TitleBar>
+                <ListItem>
+                    <Button>Item 1</Button>
+                </ListItem>
+                <ListItem>
+                    <Button>Item 2</Button>
+                </ListItem>
+                <ListItem>
+                    <Button>Item 2</Button>
+                </ListItem>
+                <div className={classNames("red")}>
```

Comment:
```suggestion
                <div className="red"}>
```

`classNames` is a utility for merging multiple classnames together and is especially useful when conditionally adding classes based on a boolean. It's not needed if the `className` is static.

Reply:
Removed.